api/messages.proto

```diff
@@ -28,15 +29,25 @@ import "google/protobuf/struct.proto";
 // of properties. Open Match stores the Ticket in state storage and enables an
 // Assignment to be associated with this Ticket.
 message Ticket {
+  // Metadata contains useful stats about the ticket.
+  message Metadata {
```

Comment:
Could be another indexable property than a separate object on the Ticket? If we do provide this functionality - we would need to provide the ability to filter tickets which have a wait time longer than x - may be?